docs/topics/templates.txt

```diff
@@ -404,6 +404,46 @@ adds defaults that differ from Jinja2's for a few options:
 * ``'auto_reload'``: ``settings.DEBUG``
 * ``'undefined'``: ``DebugUndefined if settings.DEBUG else Undefined``
 
+The default configuration is purposefully kept to a minimum. The ``Jinja2``
+backend doesn't create a Django-flavored environment. It doesn't know about
+Django context processors, filters and tags. In order to use Django-specific
+APIs, you must configure them into the environment.
+
+For example, you can create ``myproject/jinja2.py`` with this content::
+
+    from django.contrib.staticfiles.storage import staticfiles_storage
+    from django.core.urlresolvers import reverse
+
+    from jinja2 import Environment
+
+
+    def environment(**options):
+        env = Environment(**options)
+        env.globals.update({
+            'static': staticfiles_storage.url,
+            'url': reverse,
+        })
+        return env
+
+and set the ``'environment'`` option to ``'myproject.jinja2.environment'``.
+
+Then you could use the following constructs in Jinja2 templates:
+
+.. code-block:: html+jinja
+
+    <img src="{{ static('path/to/company-logo.png') }}" alt="Company Logo">
+
+    <a href="{{ url('admin:index') }}">Administration</a>
+
+The concepts of tags and filters exist both in the Django template language
+and in Jinja2 but they're used very differently. Attempting to use the same
+tags and filters in both languages would result in non-idiomatic code. Since
```

Comment:
I wanted to say that either the Django or the Jinja2 templates would look funny if you transliterated code from one into the other. I'll just drop this sentence.